packages/frameworks/src/frameworks.ts

```diff
@@ -205,6 +207,8 @@ export const frameworks = [
     ignoreRuntimes: ['@vercel/node'],
     detectors: {
       every: [
+        // Intentionally does not detect a package name
+        // https://github.com/vercel/vercel/pull/7761
```

Comment:
Because we have a [Remix builder](https://github.com/vercel/vercel/tree/main/packages/remix), we can still probably get version info out of it. Builders can optionally return identified versions.